charts/consul/Chart.yaml

```diff
@@ -15,7 +15,7 @@ annotations:
     - name: consul
       image: hashicorp/consul:1.13.1
     - name: consul-k8s-control-plane
-      image: hashicorp/consul-k8s-control-plane:0.47.1
+      image: hashicorp/consul-k8s-control-plane:0.48.0
     - name: envoy
       image: envoyproxy/envoy:v1.22.4
```

Comment:
could we update envoy image to 1.23.x? We downgraded because of this issue but looks like it was fixed in 1.23.1: https://github.com/envoyproxy/envoy/issues/22261